Xamarin.Forms.Platform.Android/Renderers/TimePickerRenderer.cs

```diff
@@ -2,24 +2,24 @@
 using System.ComponentModel;
 using Android.App;
 using Android.Content;
-using Android.Content.Res;
 using Android.Util;
 using Android.Widget;
 using Android.Text.Format;
-using ADatePicker = Android.Widget.DatePicker;
 using ATimePicker = Android.Widget.TimePicker;
 using Object = Java.Lang.Object;
 using Android.OS;
-using Xamarin.Forms.PlatformConfiguration.AndroidSpecific;
 
 namespace Xamarin.Forms.Platform.Android
 {
 	public class TimePickerRenderer : ViewRenderer<TimePicker, EditText>, TimePickerDialog.IOnTimeSetListener
 	{
 		AlertDialog _dialog;
 		TextColorSwitcher _textColorSwitcher;
-		bool _is24HourFormat;
-		string _timeFormat;
+
+		bool Is24HourView
+		{
+			get => (DateFormat.Is24HourFormat(Context) && Element.Format == (string)TimePicker.FormatProperty.DefaultValue) || Element.Format == "HH:mm";
```

Comment:
This (now renamed properly and set to a private property) was being used to determine whether the time picker dialog should open up in 24h format or not. Previously, the logic checked system settings only, ignoring user settings. Should Forms make sure system settings override developer settings or vice versa? I assume Google is okay with code overriding system settings because, after all, why does the `TimePickerDialog` API ask for a boolean for the hour format?

Suppose user code is setting format to "HH:mm" but 24h setting is set to off by the device. What's the expectation here? The changes I made allow users to bypass system settings. If the default value, `t`, is used, then we rely exclusively on system settings.

Line 21 is checking two things: default scenario where `Format` is not set and a custom scenario.